paimon-format/pom.xml

```diff
@@ -298,6 +298,10 @@ under the License.
                                     <include>org.apache.parquet:parquet-format-structures</include>
                                     <include>org.apache.parquet:parquet-jackson</include>
                                     <include>commons-pool:commons-pool</include>
+
+                                    <!-- compress -->
+                                    <include>com.github.luben:zstd-jni</include>
+                                    <include>org.xerial.snappy:snappy-java</include>
```

Comment:
Do we need snappy? I think hadoop jars already have snappy?

Reply:
Yes, I just think we should not rely on the hadoop jars. Removed now, we can add back if necessary.